app/src/main/java/com/summer/practice/tvtracker/Registration.kt

```diff
@@ -0,0 +1,12 @@
+package com.summer.practice.tvtracker
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+
+class Registration : AppCompatActivity() {
```

Comment:
please rename this activity into RegisterActivity/RegistrationActivity. We need to keep the name consistent, all activities should have Activity text in its name